plugins/robothor_plugin/robothor_environment.py

```diff
@@ -262,7 +262,7 @@ def currently_reachable_points(self) -> List[Dict[str, float]]:
         """List of {"x": x, "y": y, "z": z} locations in the scene that are
         currently reachable."""
         self.controller.step(action="GetReachablePositions")
-        return self.last_action_return
+        return self.controller.last_event.metadata["reachablePositions"]
```

Comment:
Why make this change? We should also have an `assert self.last_action_success` after the step.